packages/demo/src/app/home/home.component.scss

```
@@ -83,19 +84,57 @@
     }
   }
 
-  ul,
-  ol {
-    list-style: none;
-    padding-left: 0;
-
-    > li {
-      margin-bottom: 1rem;
-    }
-  }
-
   .bg-white {
     > *:last-child {
       margin-bottom: 0;
     }
   }
 }
+
+.accordion-button {
+  @include post.font-curve(post.$accordion-font-curve);
+
+  small {
+    @include post.font-curve('tiny');
+    display: inline-block;
+    padding-left: 1em;
+    font-weight: post.$font-weight-base;
+  }
+}
+
+.highlight {
+  display: inline;
+  font-family: post.$font-family-monospace;
+  color: post.$coral-bright;
+}
+
+.info {
+  display: block;
+  margin-bottom: 1rem;
+  font-size: post.$font-size-tiny;
+  color: post.$gray-40;
+
+  + .info {
+    margin-top: -1rem;
+  }
+}
+
+.form-check-label {
```

```suggestion
.bubble-tea .form-check-label {
```

These styles also apply for the switches which should not be grayed out when active.